### ISSUE TITLE
Input Error occured when import genesis

### ISSUE THREAD

**Issue by xiaopengyou0000** [Genesis-Embodied-AI/Genesis#51]
Exception has occurred: ValueError
numpy.dtype size changed, may indicate binary incompatibility. Expected 96 from C header, got 88 from PyObject
  File "/home/xxx/project/gennesis-world/test_genesis_world.py", line 16, in <module>
    import genesis as gs
ValueError: numpy.dtype size changed, may indicate binary incompatibility. Expected 96 from C header, got 88 from PyObject

**Comment by lupinjia** [comment]
In my case, this error is caused by `tetgen` package, which imports _tetgen in its `__init__.py`.

**Comment by nchuly** [comment]
same problem

**Comment by xiaopengyou0000** [comment]
Problem solved:
 The numpy version in this project is 1.26.4

**Comment by xiaopengyou0000** [comment]
You just need to reinstall numpy,  pip install numpy=1.26.4, also same problem for tetgen, genesis-world 0.2.0 requires tetgen==0.6.4.

**Comment by zhouxian** [comment]
yeah sorry about that. numpy 2.0 introduced a bunch of compatibilty issues that would make some of our dependencies break...

**Comment by ChengYaofeng** [comment]
> Problem solved: The numpy version in this project is 1.26.4

I tried numpy==1.25.0, this version is ok, just try `pip install numpy==1.25.0`
